modules/ztp-du-host-networking-requirements.adoc

```diff
@@ -0,0 +1,83 @@
+= Distributed unit host networking requirements
+
+The following tables provide a high level overview of the networking information and custom resources required by Red Hat Advanced Cluster Management (ACM) to provision a DU bare-metal host:
+
+.Required `AgentClusterInstall` networking fields
+[cols="2,4", width="90%", options="header"]
+|====
+|Field
+|Description
+
+|`clusterNetwork`
+| Used to allocate an IPv4 or IPv6 IP address to each node. Ensure there is no overlap with `serviceNetwork`.
+
+|`serviceNetwork`
+| Block of IPv4 or IPv6 IP addresses used for cluster services internal communication in {product-title}.  Ensure there is no overlap with `clusterNetwork`.
+
+|`machineNetwork`
+| Represents the network range for external communication. Also used to determine the API and Ingress VIP addresses for provisioning the cluster.
+|====
+
+[NOTE]
+====
+You do not need to specify API and Ingress VIP addresses for DU single node clusters. Instead, when the host is provisioned by the assisted installer service, the `machineNetwork` field in the `AgentClusterInstall` CR is used to determine the API and Ingress VIP addresses.
+====
+
+.Required `ClusterDeployment` networking fields
+[cols="2,4", width="90%", options="header"]
+|====
+|Field
+|Description
+
+|`baseDomain`
+|Base domain for the hub cluster managing the individual DU single node clusters
+|====
+
+.Required `BareMetalHost` networking fields
+[cols="2,4", width="90%", options="header"]
+|====
+|Field
+|Description
+
+|`bmc`
+|BMC address and BMC username and password credentials. See link:https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#baremetalhost-spec[BareMetalHost spec] for the list of BMC address formats for your specific DU host hardware.
+
+|`bootMACAddress`
+|Boot MAC address for the bare-metal host
+
+|`bmac.agent-install.openshift.io/hostname`
+|Optional. Configures the cluster hostname. If this field is not used, a hostname is allocated by the cluster DHCP server.
+|====
+
+.Required `NMStateConfig` networking fields
+[cols="2,4", width="90%", options="header"]
+|====
+|Field
+|Description
+
+|`mac-address`
+|Target bare-metal host MAC address. Must match the MAC address specified in the `BareMetalHost` CR.
+
+|`ip-address`
+|Target bare-metal host static IP address
+
+|`public-network-prefix`
+|bare-metal host static IP address subnet
+
+|`dns-resolver`
+|Target bare-metal host DNS server
+
+|`gateway`
+|Target bare-metal host gateway
+|====
+
+[NOTE]
+====
+`NMStateConfig` is an optional resource that allows you to configure network bonding for a pair of NICs, use a concrete VLAN, or to declare a static IP for the DU host. Each `NMState` profile has a one to one relationship with a related `InfraEnv` ISO profile used for installing {product-title} on the host. If used, `NMStateConfig` must be created before the `ClusterDeployment` resource. The `NMStateConfig` resource is not required if DHCP is enabled for the cluster network.
```

Comment:
> ! allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

```suggestion
`NMStateConfig` is an optional resource that allows you to configure network bonding for a pair of NICs, to use a concrete VLAN, or to declare a static IP for the DU host. Each `NMState` profile has a one-to-one relationship with a related `InfraEnv` ISO profile used for installing {product-title} on the host. If used, the `NMStateConfig` resource must be created before the `ClusterDeployment` resource. The `NMStateConfig` resource is not required if DHCP is enabled for the cluster network.
```